src/main/resources/view/DarkTheme.css

```diff
@@ -97,14 +101,15 @@
     -fx-label-padding: 0 0 0 0;
     -fx-graphic-text-gap : 0;
     -fx-padding: 0 0 0 0;
+    -fx-text-fill: white;
 }
 
 .list-cell:filled:even {
-    -fx-background-color: #3c3e3f;
+    -fx-background-color: #3c3e3f !important;
```

Comment:
hmm, what happens if we remove `!important` here? 🤔

Reply:
It is a quick fix for AutoComplete. So without it, we must find the cause of why the first row was different colouring

![image](https://user-images.githubusercontent.com/14960144/95738841-728a8e00-0cbc-11eb-8421-994cfb14a51c.png)